vendor/bat-native-ledger/include/bat/ledger/public/interfaces/ledger.mojom

```diff
@@ -284,7 +284,8 @@ enum UrlMethod {
   GET = 0,
   PUT = 1,
   POST = 2,
-  PATCH = 3
+  PATCH = 3,
+  DEL = 4
```

Comment:
Curious: why `DEL` instead of `DELETE`?

Reply:
`DELETE` is a macro defined for Windows. Was causing conflicts.

Reply:
Ah! Ideally these would be `kGet`, `kDelete`, but this works. Thanks!